src/cmds/qsub.c

```diff
@@ -3038,6 +3052,8 @@ make_argv(int *argc, char *argv[], char *line)
 		free(argv[i]);
 		argv[i++] = NULL;
 	}
+	if (buf_allocated)
```

Comment:
You don't need buf_allocated if you change the conditional to:
```suggestion
	if (buffer != static_buffer)
```

Reply:
Thanks @mike0042! I've addressed your comments. Please have a look at the recent commit.